Source/WebKit/Shared/RemoteLayerTree/RemoteLayerTreeTransaction.mm

```diff
@@ -282,12 +282,6 @@ static void dumpChangedLayers(TextStream& ts, const LayerPropertiesMap& changedL
     if (m_pageScaleFactor != 1)
         ts.dumpProperty("pageScaleFactor"_s, m_pageScaleFactor);
 
-#if PLATFORM(MAC)
```

Comment:
Not super urgent, but it'd be nice to be able to log the Page/MainFrame objects (there's `RemoteLayerTree` logging that prints the transactions as they arrive currently).